### PR TITLE
Add deterministic colors… and magic ⚔⛱

### PR DESCRIPTION
**Pull request by whmii**

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -1,32 +1,64 @@
...
   get '/' do
-    generator = ColorGenerator.new
+   redirect "/#{rand(10000)}"
+  end
 
-    @body_color = present(generator.body_color)
-    @body_detail_color = present(generator.body_detail_color)
-    @skin_color = present(random_skin_tone)
+  get '/:seed' do
+    srand params[:seed].to_i
```
Comment (mike-burns) [inline app.rb:11]: Clever.

File: app.rb [hunk 2]
```diff
@@ -1,32 +1,64 @@
 Bundler.require
 require 'yaml'
-
-require_relative './color_generator'
+require 'color/rgb/contrast'
 
 class App < Sinatra::Base
   get '/' do
-    generator = ColorGenerator.new
+   redirect "/#{rand(10000)}"
```
Comment (Euraldius) [inline app.rb]: Maybe use UUIDs instead of random integers?

File: app.rb [hunk 3]
```diff
@@ -1,32 +1,64 @@
...
+    Color::RGB.new(red, green, blue)
+  end
+
+ def generate_random_color
+    random_red = (0...255).to_a.sample
+    random_green = (0...255).to_a.sample
+    random_blue = (0...255).to_a.sample
+
+    Color::RGB.new(random_red, random_green, random_blue)
+
```
Comment (ghost) [inline app.rb]: extra line?

**Comment by ghost** [comment]
Looks good!!!! 🌴 ☀️ 🌊

**Pull request merged**